packages/transport/src/core/Transport.ts

```diff
@@ -59,6 +63,8 @@ export class Transport {
         allowIdentityCreation: true,
         supportedDatawalletVersion: 1,
         supportedIdentityVersion: -1,
+        supportedMinBackboneVersion: 5,
+        supportedMaxBackboneVersion: 5,
```

Comment:
don't know what to put here

Reply:
5 seems fine, we don't know if we will be compatible with v6